shared/fs/filepreview/view-container.js

```diff
@@ -37,12 +37,13 @@ const mapDispatchToProps = (dispatch: Dispatch, {path}: Props) => ({
   loadMimeType: () => dispatch(FsGen.createMimeTypeLoad({path})),
 })
 
-const mergeProps = ({_serverInfo, mimeType, isSymlink}, {loadMimeType}, {path}) => ({
+const mergeProps = ({_serverInfo, mimeType, isSymlink}, {loadMimeType}, {path, routePath}) => ({
   url: Constants.generateFileURL(path, _serverInfo),
   mimeType,
   isSymlink,
   path,
   loadMimeType,
+  routePath,
```

Comment:
What happened here? Why did this pass flow?

Reply:
I'm also a bit confused why flow didn't catch it. @chrisnojima any ideas?

Reply:
What does coverage say. Likely ownprops is any?

Reply:
Hrm I thought we didn't need to type ownprops anymore.